test/general/implementation_details/device_copyable.pass.cpp

```diff
@@ -204,9 +205,18 @@ test_device_copyable()
     //__gen_set_balanced_path
     static_assert(sycl::is_device_copyable_v<oneapi::dpl::__par_backend_hetero::__gen_set_balanced_path<
                       oneapi::dpl::__par_backend_hetero::__set_intersection,
+                      oneapi::dpl::__par_backend_hetero::__get_bounds_simple,
                       binary_op_device_copyable>>,
                   "__gen_set_balanced_path is not device copyable with device copyable types");
 
+    //__partition_set_balanced_path_submitter
+    static_assert(sycl::is_device_copyable_v<oneapi::dpl::__par_backend_hetero::__partition_set_balanced_path_submitter<
```

Comment:
Let's add the guards introduced in https://github.com/uxlfoundation/oneDPL/pull/2414 around both static asserts for the submitters.

Reply:
done, thanks